Int32 index
Int32 mot1_enc_steps
Int32 mot2_enc_steps